docs/spec/spec.md

```diff
@@ -18,83 +18,77 @@ The conforming implementation of the specification is released and included in t
 
 ```ballerina
 import ballerina/io;
 import ballerina/soap;
+import ballerina/mime;
   
-public function main () {
+public function main () returns error? {
 
-    soap:Soap12Client soapClient = new("http://ws.cdyne.com/phoneverify/phoneverify.asmx?wsdl");
+    soap:Client soapClient = check new("http://ws.cdyne.com/phoneverify/phoneverify.asmx?wsdl");
 
     xml body = xml `<quer:CheckPhoneNumber xmlns:quer="http://ws.cdyne.com/PhoneVerify/query"> 
          <quer:PhoneNumber>18006785432</quer:PhoneNumber>
          <quer:LicenseKey>0</quer:LicenseKey>
       </quer:CheckPhoneNumber>`;
 
-    var response = soapClient->sendReceive(body);
-    if (response is soap:SoapResponse) {
-        io:println(response["payload"]);
-    } else {
-        io:println(response.message());
-    }
+    xml|mime:Entity[] response = check soapClient->sendReceive(body);
+    io:println(response);
 }
 ```
 
-Follow the steps below to run this example.
-
-1. Save the example in a Ballerina file (e.g., `soapExample.bal`).
-2. Execute the `ballerina run soapExample.bal` command to run the file.
-   You will get a response similar to the following.
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<soap:Body xmlns:soap="http://www.w3.org/2003/05/soap-envelope">
-   <CheckPhoneNumberResponse xmlns="http://ws.cdyne.com/PhoneVerify/query">
-      <CheckPhoneNumberResult>
-         <Company>Toll Free</Company>
-         <Valid>true</Valid>
-         <Use>Assigned to a code holder for normal use.</Use>
-         <State>TF</State>
-         <RC />
-         <OCN />
-         <OriginalNumber>18006785432</OriginalNumber>
-         <CleanNumber>8006785432</CleanNumber>
-         <SwitchName />
-         <SwitchType />
-         <Country>United States</Country>
-         <CLLI />
-         <PrefixType>Landline</PrefixType>
-         <LATA />
-         <sms>Landline</sms>
-         <Email />
-         <AssignDate>Unknown</AssignDate>
-         <TelecomCity />
-         <TelecomCounty />
-         <TelecomState>TF</TelecomState>
-         <TelecomZip />
-         <TimeZone />
-         <Lat />
-         <Long />
-         <Wireless>false</Wireless>
-         <LRN />
-      </CheckPhoneNumberResult>
-   </CheckPhoneNumberResponse>
-</soap:Body>
+#### 2.1.3 SendOnly
+
+The `sendOnly` function fires and forgets requests. It sends the request without the possibility of any response from the service (even an error).
+
+```ballerina
+import ballerina/io;
+import ballerina/soap;
+import ballerina/mime;
+  
+public function main () returns error? {
+
+    soap:Client soapClient = check new("http://ws.cdyne.com/phoneverify/phoneverify.asmx?wsdl");
+
+    xml body = xml `<quer:CheckPhoneNumber xmlns:quer="http://ws.cdyne.com/PhoneVerify/query"> 
+         <quer:PhoneNumber>18006785432</quer:PhoneNumber>
+         <quer:LicenseKey>0</quer:LicenseKey>
+      </quer:CheckPhoneNumber>`;
+
+    _ = check soapClient->sendOnly(body);
+}
 ```
+
+## 3. Supported SOAP Versions
```

Comment:
This section has not been included into table-of-content.

Reply:
Added.